lib/dpul_collections/indexing_pipeline/figgy/transformation_consumer.ex

```diff
@@ -49,6 +49,24 @@ defmodule DpulCollections.IndexingPipeline.Figgy.TransformationConsumer do
   @impl Broadway
   @spec handle_message(any(), any(), %{required(:cache_version) => integer()}) ::
           Broadway.Message.t()
+  def handle_message(
+        _processor,
+        message = %Broadway.Message{
+          data: hydration_cache_entry = %{data: %{"metadata" => %{"deleted" => true}}}
+        },
+        %{cache_version: _cache_version}
+      ) do
+    solr_doc = Figgy.HydrationCacheEntry.to_solr_document(hydration_cache_entry)
+    marker = CacheEntryMarker.from(message)
+
+    message
+    |> Message.put_data(%{
+      marker: marker,
+      incoming_message_data: hydration_cache_entry,
+      handled_data: solr_doc
+    })
+  end
+
   def handle_message(
```

Comment:
Since they're so similar, it would be nice to have a comment saying why we are keeping both. I'm thinking it's so we can filter out things that aren't ephemera folders?

Reply:
@hackartisan I previously removed that transformation consumer function. No changes made to this file in the PR currently.

Reply:
weird, browser stuff I guess. Okay!